lib/backend/test/suite.go

```diff
@@ -525,54 +525,54 @@ func (s *BackendSuite) WatchersClose(c *check.C) {
 }
 
 // Locking tests locking logic
-func (s *BackendSuite) Locking(c *check.C) {
+func (s *BackendSuite) Locking(c *check.C, b backend.Backend) {
```

Comment:
I found this to be the least invasive to the rest of the tests.
The reason to pass the backend as the parameter is inherent danger of accessing the backend (`B` attribute on the suite) from multiple goroutines.